pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -399,6 +399,14 @@ message CommandSubscribe {
 
     // The consumer epoch, when exclusive and failover consumer redeliver unack message will increase the epoch
     optional uint64 consumer_epoch = 19;
+
+    enum IsolationLevel {
```

Comment:
@codelipenghui Should we increase `ProtocolVersion`?

Reply:
Yes. The broker should only read the isolation type from a client version that has isolation-level support. Otherwise, the broker will get an incompatibility issue.

We should also add an integration test to https://github.com/apache/pulsar/tree/master/tests/integration
And we should also add a compatibility test like https://github.com/apache/pulsar/tree/master/tests/bc_2_6_0